toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -54,6 +72,8 @@ while read -r package || [ -n "$package" ]; do
     install_one_toolchain_rpm "$package"
 done < "$packages"
 
+record_timestamp "finish adding RPM to worker chroot"
```

Comment:
Nit-pick: plural "RPMs" and let's be consistent with the message, so start all with either lower or upper case.

Reply:
Similar comment for other `record_timestamp` calls below.